apps/common/utils/toWagmiProvider.tsx

```diff
@@ -1,24 +1,24 @@
-import {toAddress, toWagmiAddress} from '@yearn-finance/web-lib/utils/address';
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
 import {ETH_TOKEN_ADDRESS, ZERO_ADDRESS} from '@yearn-finance/web-lib/utils/constants';
 import {isTAddress} from '@yearn-finance/web-lib/utils/isTAddress';
 import {assert} from '@common/utils/assert';
 
 import type {Connector} from 'wagmi';
-import type {TAddress, TAddressWagmi} from '@yearn-finance/web-lib/types';
+import type {TAddress} from '@yearn-finance/web-lib/types';
 import type {defaultTxStatus} from '@yearn-finance/web-lib/utils/web3/transaction';
 import type {GetWalletClientResult} from '@wagmi/core';
 
 export type TWagmiProviderContract = {
 	walletClient: GetWalletClientResult,
 	chainId: number,
-	address: TAddressWagmi,
+	address: TAddress,
 }
 export async function toWagmiProvider(connector: Connector | undefined): Promise<TWagmiProviderContract> {
 	assert(connector, 'Connector is not set');
 
 	const signer = await connector.getWalletClient();
 	const chainId = await connector.getChainId();
-	const address = toWagmiAddress(signer.account.address);
+	const {address} = signer.account;
```

Comment:
Are we certain that `address` is always there and that `signer` always has an `account`?